pkgs/misc/vscode-extensions/vscodeEnv.nix

```diff
@@ -0,0 +1,40 @@
+#use vscodeWithConfiguration and vscodeExts2nix to create vscode exetuable that when exits(vscode) will update the mutable extension file, which is imported when getting evaluated by nix.
+{ pkgs                            ? import <nixpkgs> {}
```

Comment:
Don't really see the point of re-importing `nixpkgs` here, most of the things here are already given (besides `vscode-utils.extensionsFromVscodeMarketplace`) which could be easily added.

Reply:
@eadwu sorry this was the wrong branch look at PR https://github.com/NixOS/nixpkgs/pull/76427 nixpkgs was removed there.